codespell_lib/data/dictionary.txt

```diff
@@ -7141,13 +7224,15 @@ executs->executes
 executute->execute
 execututed->executed
 execututes->executes
+execv->execve
```

Comment:
Technically `execv` could be a valid function. I suggest to remove this entry

See https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.basetrf1/exec.htm